proxy/pom.xml

```diff
@@ -615,6 +615,16 @@
       <artifactId>opentelemetry-exporters-jaeger</artifactId>
       <version>0.4.1</version>
       <scope>compile</scope>
+      <exclusions>
```

Comment:
???

Reply:
Added exclusions here since was receiving build errors in system code without